resources/views/register.blade.php

```diff
@@ -2,7 +2,7 @@
 <div class="container form">
         <div class="col-4">
             <h1>Registriraj se</h1>
-            <form method="post" action="register">
+            <form method="post" action="roomOption">
```

Comment:
Same as above - action for this one should be "register". The action attribute tells us which route this form will send the data to.